common/lib/capa/capa/templates/designprotein2dinput.html

```diff
@@ -11,7 +11,7 @@
     <input type="hidden" name="input_${id}" id="input_${id}" aria-describedby="answer_${id}" value="${value|h}"/>
```

Comment:
FYI, this is the protein builder. http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/protein_builder.html